### PR TITLE
[fix bug 1225896] Adjust mobile nav menu aria state timing

### PR DESCRIPTION
**Pull request by alexgibson**
This is a small change to make functional testing easier (see: https://github.com/mozilla/bedrock/pull/3575#discussion_r45218569)

### CODE REVIEW

**Comment by alexgibson** [comment]
@davehunt - does this change work for you ok?

**Comment by davehunt** [comment]
@alexgibson yep this is now passing 100% of the time, thanks!

**Comment by alexgibson** [comment]
Great, @jpetto r just to make sure I haven't fudged anything else here?

File: media/js/base/nav-main-resp.js [hunk 1]
```diff
@@ -414,11 +414,9 @@ NavMain.openSmallMenu = function()
...
     }
 
-    $('#nav-main-menu')
-        .slideDown(150)
-        .removeAttr('aria-hidden');
-
-    NavMain.toggleButton.addClass('open').attr('aria-expanded', true);
+    $('#nav-main-menu').slideDown(150, function() {
+        NavMain.toggleButton.addClass('open').attr('aria-expanded', true);
+    }).removeAttr('aria-hidden');
```
Comment (alexgibson) [inline media/js/base/nav-main-resp.js]: Ah, yes good suggestion. We don't need it for this test directly, but I think it makes sense to make sure both functions match in terms of timing.

**Comment by alexgibson** [comment]
Thanks @jpetto - updated

**Pull request merged**